server.js

```diff
@@ -21,7 +14,23 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Netflix data!");
+});
+
+app.get('/movies', (req, res) => {
+  res.status(200).json(netflixData);
+});
+
+app.get("/movies/id/:id", (req,res) => {
```

Comment:
Here you have double quotation marks, but in line 20 you have single. For consistency maybe you should consider to stick to one structure.